pokemon_scanner.py

```diff
@@ -1,41 +1,40 @@
-# scanner scans r/pokemon for mentions of pokemon
 import praw
 import dotenv
 from os import getenv
 
-### global variables below this line ###
-
 dotenv.load_dotenv()
 
-client_secret = getenv("SECRET_TOKEN")
+client_secret = getenv("CLIENT_SECRET")
 username = getenv("NAME")
-pwd = getenv("PWD")
+pwd = getenv("PASSWORD")
 client_id = getenv("CLIENT_ID")
+user_agent = getenv("USER_AGENT")
```

Comment:
It's good to pull this out to be a global variable, but probably not something the user should be changing at runtime.  `USER_AGENT` should be specific to the software that's running; so `Rock_Bot v1.3 (by u/OmnickBoy)` and the version changes with each release.  But we don't want a user to just put in some random string there.

Reply:
This was an area I wasn't sure about exactly. Since the user needs to create  their own script application on Reddit and name it, wouldn't they need to use a user_agent that matches that? Since they obviously aren't using my personal account. Maybe I'm misunderstanding how Reddit's applications can be shared.

Reply:
yeah, i get how that's confusing - it's kinda a misnomer.  `USER_AGENT` means the agent who is working on behalf of the user; so more like the user's (possessive) agent.  It identifies the software, not the person.  The username and password used to authenticate with reddit will also go with the request and that will identify the person.  

e.g. the USER_AGENT for your chrome browser is of the form `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36`.  That user agent is used every time you sign in to google, reddit, amazon, etc etc - but your username and password identify you as you.

Reply:
Ah, okay. In that case I'll make a new application called Pokemon_Bot so it's less confusing.